python/ray/data/tests/test_all_to_all.py

```diff
@@ -182,6 +176,65 @@ def test_unique_with_nulls(ray_start_regular_shared, batch_format):
     assert len(ds3.unique("col3")) == 3
 
 
+@pytest.mark.parametrize("batch_format", ["pandas", "pyarrow"])
+def test_unique_with_nulls_ignored(ray_start_regular_shared, batch_format):
```

Comment:
nit, can you parameterize `ignore_nulls` as most code is identical with the previous test?

Reply:
Sure, I can take a look (pending on the other comment whether we want to keep `ignore_nulls`)